test_shaders.py

```diff
@@ -406,6 +410,8 @@ def validate_shader_hlsl(shader, force_no_external_validation, paths):
         test_glslang = False
     if '.fxconly.' in shader:
         test_glslang = False
+    if '.task' in shader or '.mesh' in shader:
+        test_glslang = False
```

Comment:
`glslangValidator` has no support for HLSL mesh shader, so I've used `shader-playground`, for now